### PR TITLE
Fix Runtime problem

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

File: src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java [hunk 1]
```diff
@@ -47,6 +47,9 @@
     // Tag thumbnail.
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java:47]: Would be better Image thumbnail

File: src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java [hunk 2]
```diff
@@ -77,18 +80,21 @@ public FileResponse convert(Param param) throws ConvertException, CommandValueEx
         try {
             commandString = commandImageContext.buildCommand();
 
-            Runtime.getRuntime().exec(commandString);
+            Process process1 = Runtime.getRuntime().exec(commandString);
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java]: Change process1 for processThumbnail

File: src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java [hunk 3]
```diff
@@ -77,18 +80,21 @@ public FileResponse convert(Param param) throws ConvertException, CommandValueEx
...
+            process1.waitFor();
+
 
             if (!commandThumbnailList.isEmpty()) {
                 ContextStrategy commandThumbnailContext = new ContextStrategy(commandThumbnailList);
                 commandString = commandThumbnailContext.buildCommand();
-                Runtime.getRuntime().exec(commandString);
+                Process process2 = Runtime.getRuntime().exec(commandString);
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java]: Improve variable name process2

File: src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java [hunk 4]
```diff
@@ -77,18 +80,21 @@ public FileResponse convert(Param param) throws ConvertException, CommandValueEx
...
+                process2.waitFor();
             }
+            zipFile(imageParam);
 
             fileResponse.setName(imageParam.getOutputName());
             fileResponse.setStatus(MessageResponse.SUCCESS200.getMessageResponse());
-            fileResponse.setDownload(imageParam.getOutputPathFile()+imageParam.getOutputName());
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java:90]: Space in plus simbol imageParam.getOutputPathFile()+imageParam.getOutputName()
imageParam.getOutputPathFile() + imageParam.getOutputName()

File: src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java (hunk 1 above)
Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java:47]: It is only a tag name, Is it necessary to change it?

File: src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java (hunk 4 above)
Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java:90]: This code line was deleted

**Pull request merged**